src/flag_gems/fused/cross_entropy_loss.py

```diff
@@ -9,6 +9,12 @@
 from ..utils import libentry
 from ..utils import triton_lang_extension as tle
 
+REDUCTION = {
+    "none": 0,
+    "mean": 1,
+    "sum": 2,
+}
+
```

Comment:
I suggest using torch's torch/nn/_reduction.py like this

https://github.com/pytorch/pytorch/blob/4015166e5d51bc39d5a81aa59ad49720ec2a23fe/torch/nn/functional.py#L3089C1-L3097C6